packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -313,6 +313,25 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
     return res;
   }
 
+  @override
+  Future<Map<String, UpdateUserAttributeResult>> updateUserAttributes(
+      {@required UpdateUserAttributesRequest request}) async {
+    Map<String, UpdateUserAttributeResult> res;
+    try {
+      final Map<String, dynamic> data =
+          await _channel.invokeMapMethod<String, dynamic>(
+        'updateUserAttributes',
+        <String, dynamic>{
+          'data': request != null ? request.serializeAsMap() : null,
```

Comment:
Nit: I think this is effectively the same as
```suggestion
          'data': request?.serializeAsMap(),
```